.github/workflows/firebase-hosting-merge-dev.yml

```diff
@@ -19,25 +19,25 @@ jobs:
         uses: actions/setup-node@v4
         with:
           node-version: 22
-          cache: 'pnpm'
+          cache: "pnpm"
       - name: Install dependencies
         run: pnpm install
       - name: Build
-        run: pnpm build --mode="preview"
```

Comment:
AFAIK the mode is only used here

https://github.com/KonferCA/HawkHacks-Dashboard/blob/34904b858c3183fa65a4a2f684d409298e487ed5/src/main.tsx#L16-L16

It's a bit inconvenient to set the mode with the change to the `build` command so changed the usage of `import.meta.env.MODE` to `import.meta.env.VITE_APP_ENV` which has the same purpose